### DISCUSSION TITLE
ok.ru playlis not working?

### DISCUSSION THREAD

**Discussion by HalowTV** [streamlink/streamlink#5075] [Q&A] [answered]
Hi how  ok.ru working pls 
i try like so  but is not working 
streamlink -p mpv "https://ok.ru/video/4860171848238" best --player-external-http --player-external-http-port 1102

**Comment by skulblakka** [comment, marked as answer]
Can't confirm.

```
> streamlink https://ok.ru/video/4860171848238 best -l debug
Loaded config from deprecated path, see CLI docs for how to migrate: C:\Users\XXXXXX\AppData\Roaming\streamlink\streamlinkrc
[cli][warning] Loaded config from deprecated path, see CLI docs for how to migrate: C:\Users\XXXXXX\AppData\Roaming\streamlink\streamlinkrc
[cli][debug] OS:         Windows 10
[cli][debug] Python:     3.10.9
[cli][debug] Streamlink: 5.1.2
[cli][debug] Dependencies:
[cli][debug]  certifi: 2022.12.7
[cli][debug]  isodate: 0.6.1
[cli][debug]  lxml: 4.9.2
[cli][debug]  pycountry: 22.3.5
[cli][debug]  pycryptodome: 3.15.0
[cli][debug]  PySocks: 1.7.1
[cli][debug]  requests: 2.28.1
[cli][debug]  urllib3: 1.26.13
[cli][debug]  websocket-client: 1.4.1
[cli][debug] Arguments:
[cli][debug]  url=https://ok.ru/video/4860171848238
[cli][debug]  stream=['best']
[cli][debug]  --loglevel=debug
[cli][debug]  --player="mpv.exe"
[cli][debug]  --ringbuffer-size=1073741824
[cli][info] Found matching plugin okru for URL https://ok.ru/video/4860171848238
[cli][info] Available streams: 720p (worst, best)
[cli][info] Opening stream: 720p (http)
[cli][info] Starting player: "mpv.exe"
[cli][debug] Pre-buffering 8192 bytes
[cli.output][debug] Opening subprocess: "mpv.exe" --force-media-title=https://ok.ru/video/4860171848238 -
[cli][debug] Writing stream to output
[cli][info] Player closed
[cli][info] Stream ended
[cli][info] Closing currently open stream...
```

Check your if you are using the current version, check your settings, player etc... If you still think there is an issue open a [plugin issue](https://github.com/streamlink/streamlink/issues/new?assignees=&labels=plugin+issue&template=1_plugin_issue.yml&title=plugins.plugin-name-here%3A+A+very+brief+summary+of+what%27s+broken) and fill out the details (like you have been told in the other thread). Nobody can help with "but is not working".

**Reply by HalowTV** [reply]
streamlink https://ok.ru/video/4860171848238 best -l debug
[cli][info] streamlink is running as root! Be careful!
[cli][debug] OS:         Linux-4.15.0-201-generic-x86_64-with-Ubuntu-18.04-bionic
[cli][debug] Python:     3.6.9
[cli][debug] Streamlink: 3.0.3
[cli][debug] Requests(2.27.1), Socks(1.7.1), Websocket(1.3.1)
[cli][debug] Arguments:
[cli][debug]  url=https://ok.ru/video/4860171848238
[cli][debug]  stream=['best']
[cli][debug]  --loglevel=debug
[cli][info] Found matching plugin okru for URL https://ok.ru/video/4860171848238
[cli][info] Available streams: 720p (worst, best)
[cli][info] Opening stream: 720p (http)
[cli][debug] Pre-buffering 8192 bytes
error: The default player (VLC) does not seem to be installed. You must specify the path to a player executable with --player.
[cli][info] Closing currently open stream...

**Reply by bastimeyer** [reply]
> [cli][debug] Streamlink: 3.0.3

edit:
Thanks for answering, @skulblakka
